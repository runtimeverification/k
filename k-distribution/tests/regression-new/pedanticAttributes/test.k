// Copyright (c) K Team. All Rights Reserved.
module TEST
  syntax Int ::= r"[\\+-]?[0-9]+" [prefer, token, prec(2), badAtt(10)]
               | Int "+" Int [group(badAttButOkay),badAtt,function]
endmodule
