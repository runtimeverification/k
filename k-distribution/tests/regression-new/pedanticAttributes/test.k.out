[Error] Compiler: Unrecognized attributes: [badAtt]
Hint: User-defined groups can be added with the group(_) attribute.
	Source(test.k)
	Location(3,18,3,71)
	3 |	  syntax Int ::= r"[\\+-]?[0-9]+" [prefer, token, prec(2), badAtt(10)]
	  .	                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[Error] Compiler: Unrecognized attributes: [badAtt]
Hint: User-defined groups can be added with the group(_) attribute.
	Source(test.k)
	Location(4,18,4,68)
	4 |	               | Int "+" Int [group(badAttButOkay),badAtt,function]
	  .	                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[Error] Compiler: Had 2 structural errors.
